src/objects/Course__c.object

```diff
@@ -55,7 +55,7 @@
         <label>HEDA Course Compact Layout</label>
     </compactLayouts>
     <deploymentStatus>Deployed</deploymentStatus>
-    <description>Academic course that a Department or School (the degree-granting Account type for the University) offers.</description>
```

Comment:
@juliesnow  Please note, I've updated the description fo the Course object to use the term "educational institution" instead of university.